x/examples/outline-vpn-app/fullstack_app/start.go

```diff
@@ -0,0 +1,124 @@
+package fullstack_app
+
+import (
+	"context"
+	"encoding/base64"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"github.com/Jigsaw-Code/outline-sdk/transport/shadowsocks"
+	"github.com/Jigsaw-Code/outline-sdk/x/mobileproxy"
+	"github.com/a-h/templ"
+)
+
+var APP_ADDRESS = ":8080"
+var PROXY_ADDRESS = ":8181"
+var SYSTEM_ADDRESS = ":8282"
+
+var PROXY_DISCONNECT_TIMEOUT = 5
+var SYSTEM_TUNNEL_ENDPOINT = "${SYSTEM_ADDRESS}/tunnel/${PROXY_ADDRESS}"
+
+func Start() {
+	vpn := VPNController{}
+
+	http.Handle("/", templ.Handler(serverView()))
+	http.Handle("/connection/ss://<my-shadowsocks-key>", http.HandlerFunc(vpn.handleConnectionEndpoint))
+	http.Handle("/disconnection/ss://<my-shadowsocks-key>", http.HandlerFunc(vpn.handleDisconnectionEndpoint))
+
+	http.ListenAndServe(APP_ADDRESS, nil)
+}
+
+type VPNController struct {
+	proxy  *mobileproxy.Proxy
+	tunnel io.Reader
+}
+
+func (vpn *VPNController) handleConnectionEndpoint(responseWriter http.ResponseWriter, request *http.Request) {
+	endpoint, encryptionKey := parseStaticShadowsocksAccessKey(request.URL.Path)
```

Comment:
Check for any existing connections first? But still, there might be race conditions here, we can deal with that later.

```suggestion
	if vpn.proxy != nil {
		http.Error(responseWriter, "already connected", http.StatusConflict)
		return
	}

	endpoint, encryptionKey := parseStaticShadowsocksAccessKey(request.URL.Path)
```